app/controllers/subnets_controller.rb

```diff
@@ -6,6 +6,7 @@ class SubnetsController < ApplicationController
 
   def index
     @subnets = resource_base_search_and_page([:domains, :dhcp])
+    @subnets = @subnets.network_reorder(params[:order]) if params[:order].try(:include?, 'network')
```

Comment:
Maybe this should be an exact match. `network_reorder` would return `'INET_ATON(network)'_address` if the order param was `network_address`.

Reply:
Hm, good catch. I guess there are 3 possible values I should test. 'network ASC', 'network DESC' and 'network'